demo/api/schema.gql

```diff
@@ -143,7 +143,7 @@ type Page implements DocumentInterface {
   content: PageContentBlockData!
   seo: SeoBlockData!
   createdAt: DateTime!
-  dependencies: [Dependency!]!
+  dependencies(filter: DependenciesFilterInput): [Dependency!]!
```

Comment:
For products the filter is only called ProductsFilter, we should use the same naming schema here